src/blockchain/node/manager/Manager.cpp

```diff
@@ -1063,7 +1125,7 @@ auto Base::RequestBlocks(
 
 auto Base::reset_heartbeat() noexcept -> void
 {
-    static constexpr auto interval = 5s;
+    static constexpr auto interval = 2s;
```

Comment:
These times can be change to fine tune the performance, we are not relying on delay timing to avoid race conditions. Race conditions were dealt with by design changes, including an individual reactor thread (processing loop) for each Actor or Worker.